locust/contrib/postgres.py

```diff
@@ -0,0 +1,52 @@
+from locust import TaskSet, User, events
+
+import time
+
+import psycopg
+
+
+def create_conn(conn_string):
```

Comment:
I think this method is too small to make sense!

Reply:
Good call, it looks a lot cleaner calling the `psycopg.connect` directly